ui/src/components/Property/PropertyEditror.vue

```diff
@@ -0,0 +1,211 @@
+<script setup>
```

Comment:
Added files should contain the license header

Reply:
ok